provider/go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/pulumi/pulumi-spotinst/provider/v3
 
-go 1.19
```

Comment:
Is there a reason this is downgraded, curious?

Reply:
Looks like it was altered when `go mod tidy` was ran, no good reason – will change back.

Reply:
Perhaps the toolchain you're running go mod tidy with is 1.18.